packages/connectivity/src/scp-cf/destination/destination-accessor-provider-subscriber-lookup.spec.ts

```diff
@@ -326,18 +333,20 @@ describe('JWT type and selection strategies', () => {
 });
 
 describe('call getAllDestinations with and without subscriber token', () => {
-  beforeEach(() => {
+  beforeEach(async () => {
```

Comment:
[q] Same here, await destinationCache.clear(); still needed?

Reply:
Yeah, I just missed clearing it up, done now.